app/models/bill.rb

```diff
@@ -3,4 +3,14 @@ class Bill < ApplicationRecord
 
   has_many :bookings, dependent: :destroy
   belongs_to :user
+
+  scope :find_bill, ->(user_id){where("user_id = #{user_id}")}
```

Comment:
chỗ này dùng association đi em => `user.bills`
```suggestion
  scope :find_bill, ->(user_id){where user_id: user_id}
```